src/configuration/utils.ts

```diff
@@ -16,6 +16,10 @@ export const coerceOptions = (
       AVAILABLE_VERSIONS.find(v => v.value.flavor === flavor)?.value.version ||
       AVAILABLE_VERSIONS[AVAILABLE_VERSIONS.length - 1].value.version;
   }
+  if (options.jsonData.serverless) {
+    flavor = flavor || Flavor.OpenSearch;
+    version = version || '1.0.0';
+  }
```

Comment:
so why is this an Or case? I thought in serverless was always the same?

Reply:
done, it was in the old code and I didn't think about it